app/livechat/server/lib/screenSharing/IScreenSharingProvider.ts

```diff
@@ -0,0 +1,5 @@
+export interface IScreenSharingProvider {
+	config: any;
+	getJWT(agent: any): any;
```

Comment:
This is a specific method and needs to be implemented inside the provider class when necessary.